app-admin/systemrescuecd-x86/systemrescuecd-x86-4.6.0.ebuild

```diff
@@ -18,8 +18,14 @@ S=${WORKDIR}
 RESTRICT="mirror"
 
 src_install() {
-	insinto "/usr/share/${PN%-*}"
-	doins "${DISTDIR}/${P}.iso"
+	insinto /usr/share/${PN%-*}
+	doins ${DISTDIR}/${P}.iso
+
+	exeinto /etc/grub.d
+	newexe ${FILESDIR}/systemrescuecd.grub 39_systemrescuecd
```

Comment:
Oh, and add yourself as the first `<maintainer/>`. You can add me as second to have someone to commit stuff for you ;-).